packages/www/layouts/assetDetail.tsx

```diff
@@ -86,17 +88,44 @@ const AssetDetail = ({
         onEdit={onEditAsset}
         asset={asset}
       />
-
       <EmbedVideoDialog
         isOpen={embedVideoDialogOpen}
         onOpenChange={setEmbedVideoDialogOpen}
         playbackId={asset?.playbackId}
       />
-
       <Layout id="assets" breadcrumbs={breadcrumbs}>
-        <Box css={{ px: "$6", py: "$7" }}>
+        <Box css={{ px: "$6", py: "$4" }}>
+          {asset?.status?.phase === "failed" ? (
+            <Alert variant="destructive">
+              <TriangleAlert size={18} />
+              <AlertTitle>Internal error processing file</AlertTitle>
+              <AlertDescription>{asset.status.errorMessage}</AlertDescription>
+            </Alert>
+          ) : asset?.status?.phase === "processing" ||
+            asset?.status?.phase === "uploading" ? (
+            <Alert variant="warning">
+              <Clock10 size={18} />
+              <AlertTitle>
+                <span className="capitalize">{asset.status.phase}</span> the
+                asset
+                {asset.status.progress
+                  ? ` (${asset.status.progress * 100}%)`
+                  : ""}
```

Comment:
I guess this will only be the case when the asset is uploading. If it's easy enough, could show the upload progress here as well which should be available on the frontend. Nevermind if not easy tho

Reply:
ha, yeah! sounds good!

I will add that as well :)

Reply:
Just catching up with the PRs :)

i've checked it and for now, it might be a little complicated because when the file is uploading, we don't have their asset Id, we could filter the `getFilteredFileUploads` by name, but that is not really ideal solution. 

I'm going to merge this PR, but once we completed the asset redesign, I will make sure to include it as well :)